src/uu/du/src/du.rs

```diff
@@ -257,6 +263,18 @@ fn read_block_size(s: Option<&str>) -> usize {
     }
 }
 
+fn choose_size(matches: &ArgMatches, stat: &Stat) -> u64 {
+    if matches.is_present(options::INODES) {
+        stat.inodes
+    } else if matches.is_present(options::APPARENT_SIZE) || matches.is_present(options::BYTES) {
+        stat.size
+    } else {
+        // C's stat is such that each block is assume to be 512 bytes
```

Comment:
```suggestion
> The st_blocks field indicates the number of blocks allocated to the file, 512-byte units.
```
Just c&p the relevant info, no paraphrasing needed.

Also 
> C's stat

is confusing because there's more than one `stat` command implementation written in C, e.g. if you mean the one from GNU's coreutils project, the link would be https://linux.die.net/man/1/stat.
The link you provide as reference is for the system call `stat` (https://en.wikipedia.org/wiki/Man_page#Manual_sections).

Reply:
Yeah, that's what it meant. It was refactored from the `uumain`.